store/sqlstore/channel_store.go

```diff
@@ -1412,7 +1415,7 @@ func (s SqlChannelStore) SaveMember(member *model.ChannelMember) (*model.Channel
 	return newMembers[0], nil
 }
 
-func (s SqlChannelStore) saveMultipleMembersT(transaction *gorp.Transaction, members []*model.ChannelMember) ([]*model.ChannelMember, error) {
+func (s SqlChannelStore) saveMultipleMembersT(members []*model.ChannelMember) ([]*model.ChannelMember, error) {
```

Comment:
I think the initial idea was to use it, there are some queries (read queries) before one single write query, is not a huge deal, but can lead to inconsistencies depending on the isolation level. I don't think is a huge deal, but I would try to use the transaction and load test it. It can improve the performance or generate a lot of contention depending on the isolation level, I think. For now, I think is save to remove that, and create a ticket to investigate the option of using a transation.

Reply:
Perhaps we should rename it if it's not using a transaction just to keep things clear? If I see a `somethingT` function in this code, I just assume it requires a transaction

Reply:
Agree, @mahmud2011 - let's do this.